config/manager/manager.yaml

```diff
@@ -40,4 +40,13 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
+        env:
```

Comment:
this is the "downward API" in Kubernetes - that allows to pass some Kuberentes information (pod name, namespace, annotations etc) to the container